geode-core/src/main/java/org/apache/geode/internal/size/ReflectionSingleObjectSizer.java

```diff
@@ -89,62 +90,89 @@ public static long sizeof(Class<?> clazz) {
     return sizeof(clazz, true);
   }
 
+  public static long sizeof(Class<?> clazz, boolean roundResult) {
+    return sizeof(clazz, roundResult, unsafe);
+  }
+
+  @VisibleForTesting
```

Comment:
I think you could leave off the annotation from these methods. Just have them be package-private and don't worry if calling code is product or test. It's a valid method either way.

Reply:
okay, but they were all private before adding unit test coverage

Reply:
This existing class already had no state and these new methods are called by public static methods that this class exposes. This class has a static initialize that sets the "unsafe" static field.
So it could be done but would require this entire class to be reworked to be an instance.
I don't think it is worth taking on for the jdk17 work.